EIPS/eip-4788.md

```diff
@@ -158,24 +154,29 @@ push0
 revert
 
 jumpdest
-push3 0x018000
 push0
 calldataload
-mod
 dup1
+iszero
+push1 0x49
+jumpi
+
+push3 0x001fff
+dup2
+mod
+swap1
+dup2
 sload
-push0
-calldataload
 eq
-push1 0x37
+push1 0x3c
 jumpi
 
 push0
 push0
 revert
 
 jumpdest
-push3 0x018000
+push3 0x001fff
```

Comment:
This should be push2 now.

Reply:
yeah, this was a miss. However, not worth fixing, the runtime execution gascost is identical, the only difference is that the code is one byte larger than it would have needed to be.

Reply:
This repeats 4x so 4 bytes wasted. Forever :)

Reply:
Yeah. Now there are 16 bits less left in the world.

🚻 ☮️